pulpcore/cli/container/repository.py

```diff
@@ -286,3 +287,21 @@ def copy_manifest(
         digests=digests or None,
         media_types=media_types or None,
     )
+
+
+@repository.command(allowed_with_contexts=push_container_context)
+@name_option
+@href_option
+@click.option("--digest", help=_("SHA256 digest of the Manifest file"), required=True)
+@pass_repository_context
+def remove_image(
+    repository_ctx: PulpContainerPushRepositoryContext,
+    digest: str,
+) -> None:
+    digest = digest.strip()
+    if not digest.startswith("sha256:"):
+        digest = f"sha256:{digest}"
+    if len(digest) != 71:  # len("sha256:") + 64
+        raise click.ClickException("Improper SHA256, please provide a valid 64 digit digest.")
```

Comment:
Hmm, maybe not worth the trouble, but this looks like things we usually put in the option callback.